src/refs.bib

```diff
@@ -1855,6 +1799,27 @@ @software{gwcs
   url          = {https://doi.org/10.5281/zenodo.5800080}
 }
 
+@software{synphot,
+  author       = {P. L. Lim and
```

Comment:
How was this author list generated? Vicki and some people are missing.

Reply:
From `zenodo`:  https://zenodo.org/record/5711522/export/hx

Reply:
Huh, it must have blindly copied over contributor list... Are we allowed to change author listing to deviate from Zenodo or has the ship sailed?

Reply:
Is this slightly better? I cannot tell. 🤷 

https://ui.adsabs.harvard.edu/abs/2018ascl.soft11001S/exportcitation

Reply:
It would be odd to include authors that aren't on the document that is referenced.  Keep in mind that the author list will be truncated in the paper.  Is there a better or another `synphot` reference?

Reply:
https://github.com/spacetelescope/synphot_refactor/blob/master/CITATION.md was the best I could come up with. I guess I'll just leave this to your good judgement...

Reply:
It's your package, so I'll leave it up to you.  Tell me what reference (or references) you want to use.

Reply:
Let's go with https://ui.adsabs.harvard.edu/abs/2018ascl.soft11001S/exportcitation then? Then it would be consistent with `CITATION.md` in the package and the author list is less controversial. Thanks!

Reply:
Wait a minute... how is this different from this below? Do we need two entries?

https://github.com/astropy/astropy-v5.0-paper/blob/2a62a2ee5326b969c397fb869ab56cc35cc56f0d/src/refs.bib#L1356-L1361

Reply:
No.  I removed the duplicate entries and update the `synphot` reference to the one you suggested (ASCL).